docs/Running-Mastodon/Production-guide.md

```diff
@@ -34,9 +34,17 @@ server {
   keepalive_timeout    70;
   sendfile             on;
   client_max_body_size 0;
-  gzip off;
 
-  root /home/mastodon/live/public;
+
+  gzip on;
```

Comment:
Is this tested in a live production? I remember disabling gzip on purpose due to some incompatibility very early on, however a lot has changed since then, so it might be okay now.